### PR TITLE
Fix for custom repository URL

### PR DESCRIPTION
**Pull request by nvazquez**
Fixes: https://github.com/apache/cloudstack/issues/5188

Explicitly add a variable URL to indicate the lines refer to a custom repository

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan docbuild

File: source/installguide/building_from_source.rst [hunk 1]
```diff
@@ -383,14 +383,17 @@ line:
 
 .. parsed-literal::
 
-   deb http://server.url/cloudstack/repo/binary ./
+   deb <http://server.url>/cloudstack/repo/binary ./
```
Comment (DaanHoogland) [inline source/installguide/building_from_source.rst:386]:
```suggestion
   deb http://<server.url>/cloudstack/repo/binary ./
```

File: source/installguide/building_from_source.rst [hunk 2]
```diff
@@ -383,14 +383,17 @@ line:
...
 
 If you signed your Release file with GnuPG, import the signing key
 on your target system first.
 
 .. parsed-literal::
 
-   $ wget -q -O - http://server.url/cloudstack/repo/binary/KEY.gpg | sudo apt-key add -
+   $ wget -q -O - <http://server.url>/cloudstack/repo/binary/KEY.gpg | sudo apt-key add -
```
Comment (DaanHoogland) [inline source/installguide/building_from_source.rst:393]:
```suggestion
   $ wget -q -O - http://<server.url>/cloudstack/repo/binary/KEY.gpg | sudo apt-key add -
```

File: source/installguide/building_from_source.rst [hunk 3]
```diff
@@ -383,14 +383,17 @@ line:
...
+
+.. note::
+   In the previous lines the variable <http://server.url> must be replaced with the URL of the repository
```
Comment (DaanHoogland) [inline source/installguide/building_from_source.rst:396]:
```suggestion
   In the previous lines the variable <server.url> must be replaced with the address of the repository
```

**Review by rhtyd — APPROVED** [review]

**Pull request merged**